src/spec/product-type-importer.spec.coffee

```diff
@@ -0,0 +1,113 @@
+Promise = require 'bluebird'
+ProductTypeImporter = require '../lib/product-type-import'
+{SphereClient} = require 'sphere-node-sdk'
+
+errMissingCredentials = 'Missing configuration in env variables'
+
+argv =
+  clientId: process.env.SPHERE_CLIENT_ID
+  clientSecret: process.env.SPHERE_CLIENT_SECRET
+  projectKey: process.env.SPHERE_PROJECT_KEY
```

Comment:
Usually we put credentials into a file and read it from there

https://github.com/sphereio/sphere-node-sdk/blob/master/create_config.sh

So we can also run the tests locally without env vars